core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/Group.java

```diff
@@ -22,14 +22,12 @@
 
 import java.util.List;
 
-
 /**
  * Represents a Group.
```

Comment:
Shall we add a more descriptive class description here. many may confuse Role and Group

Reply:
Updated the PR.